sched/Kconfig

```diff
@@ -755,6 +755,7 @@ endif # SCHED_CHILD_STATUS
 config SCHED_WAITPID
 	bool "Enable waitpid() API"
 	default n
+	select SCHED_HAVE_PARENT if BUILD_KERNEL
```

Comment:
should we change to `depends on`

Reply:
I tried that but it gave some syntax error. That would be better IMO yes